<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DependencyChaos </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="currentrpath">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Current rpath is &quot;loaderpath&quot;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QFrame" name="progressFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeView" name="treeView">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DropOnly</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="LogTextFrame">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAblage">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="separator"/>
    <addaction name="actionSet_current_rpath"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFix_ID"/>
    <addaction name="actionFix_Dependency"/>
    <addaction name="actionCheck_selected"/>
    <addaction name="actionSearch_and_replace_in_depend"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuAblage"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ffnen"/>
   <addaction name="separator"/>
   <addaction name="actionFix_ID"/>
   <addaction name="actionFix_Dependency"/>
   <addaction name="actionCheck_selected"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionSearch_and_replace_in_depend"/>
   <addaction name="actionSet_current_rpath"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/Analysis.png</normaloff>:/imglist/img/Analysis.png</iconset>
   </property>
   <property name="text">
    <string>Open files...</string>
   </property>
   <property name="toolTip">
    <string>Open dylib...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSearch_and_replace_in_depend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/edit-validated.png</normaloff>:/imglist/img/edit-validated.png</iconset>
   </property>
   <property name="text">
    <string>Search and replace</string>
   </property>
   <property name="toolTip">
    <string>Search and replace in dependency</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFix_ID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/edit-validated.png</normaloff>:/imglist/img/edit-validated.png</iconset>
   </property>
   <property name="text">
    <string>Fix ID</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionFix_Dependency">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/edit-validated.png</normaloff>:/imglist/img/edit-validated.png</iconset>
   </property>
   <property name="text">
    <string>Fix Dependency</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSet_current_rpath">
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/buttondown.png</normaloff>:/imglist/img/buttondown.png</iconset>
   </property>
   <property name="text">
    <string>Set current rpath</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/Refresh.png</normaloff>:/imglist/img/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCheck_selected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgres.qrc">
     <normaloff>:/imglist/img/Analysis.png</normaloff>:/imglist/img/Analysis.png</iconset>
   </property>
   <property name="text">
    <string>Check selected</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgres.qrc"/>
 </resources>
 <connections/>
</ui>
